### ISSUE TITLE
Sprint 1.33.0 2/2 - Feb 7 to Feb 18

### ISSUE THREAD

**Issue by marcushyett-ph** [PostHog/posthog#8362]
# Global Sprint Planning
* Weekly reiterating of company goals for 2022
    * Goals:
      * Quality: Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
      * Extensibility: You can instantly connect PostHog to any other system 
      * Deployments: PostHog is the easiest self-hosted product to deploy and scale in the world
    * Non Goals:
        * Build lots of new low quality / partial features
        * Build everything a single enterprise customer wants just to close a deal

## Retro: Status of Outcomes from Previous Sprint

1. **Migration of cloud to helm chart**: Staging environment for cloud live! @guidoiaquinti + @hazzadous (Status: Please update @hazzadous )
2. **Unblock Clickhouse upgrade:** 3 Key users with async migrations complete @tiina303 + @yakkomajuri. Status: All pre-release changes to async migrations were shipped and customers were contacted about this. Actually having them upgrade was blocked by the release though. (Reflection: unsure why this was a goal if the release was a natural blocker) 
3. **Performance**:~40% improvement for big clients: Events table migration on cloud complete @macobo Status: Not done - cloud helm chart migration took up most concentration, will finish it when back from vacation. Documented state + next steps in https://github.com/PostHog/posthog/issues/5684#issuecomment-1026938112 but decided _not_ to hand this off w/ guido/harry.
2. **Validate Automated Insights Feasibility**: @EDsCODE (with @neilkakkar) (Status: Creating a taxonomy can be done especially with the help of openai but translating event meanings to useful insights is still a big ?. Follow-up direction would be to automate "related" insights first where we show insights related to an insight in focus based on predetermined criteria @EDsCODE  )
3. **Get customers using experiments**: @neilkakkar (Status: Getting along! Some setbacks, but overall things looking good, users starting to create experiments  )
4. Insight/dashboards permissions (#7620) including permalink sharing with the project_id thing. Blocker for a large deal. @Twixes (Status: Permissions should be ready within the sprint)
5. Exposing event and property taxonomy. (Status: Please update @alexkim205)
6. Audit log. Heavily related to collaboration. We're going more into debt if we don't have it. @pauldambra (Status: not done. some initial investigation, but finishing datetime and then support hero got in the way  )

## Plan: Proposed Goals for Next Sprint 

Each goal should have a single owner. Owner can only be an engineer.

1. Taxonomy. New data management page. @alexkim205
1. Audit log. Capturing the data, nothing social. @pauldambra
1. Collaboration. Sharing urls with project ID @Twixes
1. Nail Trends: AND + OR filtering & Behavioural filtering (without optimizing performance) @EDsCODE 
1. Increase ingestion robustness: followups for postgres outage & dumb events service & DLQ @yakkomajuri 
1. Helm staging env - Ready to accept traffic @guidoiaquinti (@hazzadous)

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
Team ___

## Retro

<!-- Talk about what went well, what didn't go well and any actions to improve next time -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## Planning

<!-- Each item should have a single owner. Owner can only be an engineer. -->

### High priority

-

### Low priority / side quests

-

```

**Comment by mariusandra** [comment]
Team Core Experience

## Retro

Show of hands, who's tired: everybody raises hands.

- Paul (support hero): mentally very tired. Papercups is not a good UI for a support desk. 
- Alex: it was a lot. Plenty of less familiar backend work, learned a lot. Taxonomy: done a lot of work to surface definitions and in context editing. Chugging along. Collaboration: no sense to switch context for next 2 weeks, going to focus on Taxonomy. More discussion before the big refactors would have made sense. A lot of context switching.
- Rick: secondary support first week, was good to go fixing bugs... but still dragged into fixing those this week. 
- Paolo: tried to stop execution (less on github/figma), focus on creating context and a strategic approach... Has been cool. Want to find a balance.
- Chris: a lot of projects, but that's usual and fine
- Michael: got my drivers license 🚗, dashboards are getting great, permissions too. 🎉 

## Hang over items from previous sprint

- Various.

### High priority

- **Taxonomy.** New data management page. @alexkim205 
- **Audit log.** Capturing the data, nothing social. @pauldambra 
- **Collaboration**. Sharing urls with project ID @Twixes 

### Low priority / side quests

- **Dashboard UX.** Ship 100%, make sure people have a great experience. @Twixes 
- **Recordings 🐞🔫** Making recordings setup compatible with all mediums between chair and keyboard (2 instances of posthog-js, permitted urls, etc). @rcmarron 
- **Filtering UX** - https://github.com/PostHog/posthog/issues/8257 and improve general UX (filter closes when it shouldn't, etc) @mariusandra 
- **Demo** - Improved Demo environment with better test data @Twixes

**Comment by EDsCODE** [comment]
## Retro

**Automated insight validation:**
- DId taxonomy investigation
- Openai can help us discover patterns of events
- Translating events to useful insights unclear
- Good backup approach is doing automated related insights where we surface useful information based on specific criteria related to each insight

**Yeetcode:**
- Moved all the top level endpoints
- First major query PR move is in progress

**Support hero** 
- more questions now that just require context into how the user is setting up their stuff / what they’re doing wrong vs actual issues/bugs with the app  


**Experiments:**
- Some iterations on running experiment screens based on feedback
- Struggling to get users creating experiments: Something to solve for next


## Planning:

**Nail Trends:** 
- AND + OR filtering & Behavioural filtering (without optimizing performance)
     - Frontend: work with Chris to implement designs for new and+or filtering interface
     - Cohort filtering
- For insight filtering, structure the property payload to work with and+or filtering and migrate previous filter properties to be compatible
- For behavior filtering, expand cohort filtering to allow for more complex filtering
- Misc
     - https://github.com/PostHog/posthog/pull/6435



**Experimentation background tasks for increasing conversion**

**Continue Yeetcode**
